src/components/Comment.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+
+function Comment(props) {
+    return (
```

Comment:
use 2 whitespaces instead of 4
```suggestion
  return (
```